### ISSUE TITLE
How to export keystore from the wallets which created by Mnemonic Phrase

### ISSUE THREAD

**Issue by Jonyzfu14** [Boilertalk/Web3.swift#56]
We can import a wallet using Mnemonic Phrase, but how to export keystore file after this wallet generated. Shall I use keystore.swift you wrote?

**Comment by Ybrin** [comment]
Yes, use keystore.swift to export private keys encrypted as keystore files.

Btw, how do you import mnemonics? I actually need this right now...

**Comment by sche** [comment]
@koraykoska hey
I just wonder, so the lib does not support importing an account by mnemonic yet, right?

**Comment by koraykoska** [comment]
@sche No. I started an implementation a while back. But I did not finish it and it's been too long to get into it fast. If you create an IssueHunt issue I can look into it. Or you can always do it yourself and make a PR.

**Comment by koraykoska** [comment]
@sche Actually, maybe it's even better to make a custom library like keystore.swift. I like keeping this repository clean and working only with raw bytes.

**Comment by koraykoska** [comment]
@sche Some relevant information regarding this:

https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki

https://github.com/bitcoinjs/bip39

https://github.com/miguelmota/ethereum-hdwallet

Join our Telegram channel for more discussions: https://t.me/joinchat/BPk3DE6CTFaiOolSIZNLyg
